packages/ndla-ui/src/ProgrammeCard/index.tsx

```diff
@@ -0,0 +1,9 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+export { default as EducationalProgrammesCard } from './ProgrammeCard';
```

Comment:
Her bør vi bestemme oss for ett navn og ikkje to. Enten ProgrammeCard eller EducationalProgrammeCard. Men då må komponenten døpes om. Og i alle fall har korrekt fleirtallsvariant av ordet. Altså ikkje EducationalProgrammesCard.

Reply:
Gikk for ProgrammeCard.